packages/eslint-plugin-office-addins/src/utils/utils.ts

```diff
@@ -72,12 +72,28 @@ function findOfficeApiReferencesInScope(scope: Scope): void {
   scope.childScopes.forEach(findOfficeApiReferencesInScope);
 }
 
+export function isProperty(node: TSESTree.Node): boolean {
```

Comment:
This doesn't make sense . . . you aren't testing if the node it a property, you are checking if there is a property in the arguments.

Reply:
Agree, the name of the function is wrong.

Reply:
What is the purpose of checking if we are making a method call with a property argument (why does it matter)?

Reply:
I guess the problem lies mostly in the function name. I made a few adjustments to make it check for calling a method and not a property, which makes more sense.

Reply:
Just fixed it as we discussed @millerds . The PR is ready to be reviewed again